src/ArticleCard.jsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticleById } from "./utils/api";
+
+function ArticleCard() {
+  const { article_id } = useParams();
+  const [article, setArticle] = useState([]);
+
+  useEffect(() => {
+    getArticleById(article_id).then((articleData) => {
```

Comment:
Would be nice to see a loading state here and also on the main articles list component :)